app/views/last_seen_issues/open_popup.js.erb

```diff
@@ -0,0 +1,19 @@
+showModal('ajax-modal', '800px');
+$('#ajax-modal').parent().css("top", "2%").css("left", "45%");
+var issues_ids = JSON.parse(localStorage.getItem("last_seen_issues"));
+if (!issues_ids || issues_ids.length == 0) {
+    $('#ajax-modal').html(`<h1><%= t(:history_missing_message) %><h1>`).css("text-align", "center").css("min-height", "60px");
+} else {
+    $.ajax({
+        method: 'GET',
+        url: "/show_last_seen_issues?ids=" + issues_ids.join(','),
+        data: "",
+        success: function(data) {
+            $('#ajax-modal').html(data);
```

Comment:
What happens if the query takes some time to return from server? Is the popup filled with the previous values? Or is it empty? If empty, can you see why? (maybe `showModal` clears the data on close?)

Reply:
Can we add a "Loading ..." or something until it gets populated?

Reply:
Done.